.gitignore

```diff
@@ -1,6 +1,6 @@
 src/
 /tmp/*
-*.csv
+# *.csv
```

Comment:
Are we no longer ignoring this file because it is the input for the test?

Reply:
Do we want to allow all .csv files, or just the one for the test?

Reply:
Yes, will address in final commit (ignore all csv files except our "fixture")